client/src/components/all-components/imagine-components/SurveyHandlerComp.jsx

```diff
@@ -1,21 +1,29 @@
 import { React, useState } from "react";
 import { PropTypes } from "prop-types";
-import Survey from "../components/Survey";
+import Survey from "./Survey";
 import { navigate } from "@reach/router";
-import PreSurveyQuestions from "../data/preSurveyQuestions";
-import PostSurveyQuestions from "../data/postSurveyQuestions";
+import PreSurveyQuestions from "../../imagine23/data/preSurveyQuestions";
+import PostSurveyQuestions from "../../imagine23/data/postSurveyQuestions";
```

Comment:
I would add 23 to the end of these object names for clarity: `PreSurveyQuestions23` and `PostSurveyQuestions23`